app/components/attachment_button.js

```diff
@@ -328,6 +330,20 @@ export default class AttachmentButton extends PureComponent {
         }, 100);
     };
 
+    removeProfileImage = () => {
+        this.props.removeProfileImage();
+        this.props.navigator.dismissModal({
+            animationType: 'none',
+        });
+    };
+
+    isCustomProfileImage = () => {
+        if (this.props.profileImageUri.includes('?')) {
```

Comment:
Why is this needed?

Reply:
Attempting to identify the default image vs custom image URL. It looks like the only difference between the image URL returned is the query string with the last modified timestamp. There would be no need to show the "Remove Image" option if the user profile has the default image set.

Reply:
yeah I thought it would have the timestamp set to 0 but you are right here